plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/MetastoreUtil.java

```diff
@@ -324,11 +324,6 @@ public static PrincipalPrivileges buildInitialPrivilegeSet(String tableOwner)
                 ImmutableMultimap.of());
     }
 
-    public static boolean isPartitionKeyFilterFalse(TupleDomain<String> partitionKeysFilter)
-    {
-        return partitionKeysFilter.isNone() || partitionKeysFilter.getDomains().get().values().stream().anyMatch(Domain::isNone);
```

Comment:
I don't think `partitionKeysFilter.getDomains().get().values().stream().anyMatch(Domain::isNone)` is needed, see
`io.trino.spi.predicate.TupleDomain#TupleDomain` (it checks `containsNoneDomain`)

Reply:
Ah, you're right, `TupleDomain` guarantees that no domain will be `none`